### PR TITLE
[Exchanges] add hollaex support

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: tests_additional/real_exchanges/test_hollaex.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    ExchangeConstantsTickersColumns as Ectc
+from tests_additional.real_exchanges.real_exchange_tester import RealExchangeTester
+# required to catch async loop context exceptions
+from tests import event_loop
+
+# All test coroutines will be treated as marked.
+pytestmark = pytest.mark.asyncio
+
+
+class TestHollaexRealExchangeTester(RealExchangeTester):
```
Comment (Herklos) [inline tests_additional/real_exchanges/test_hollaex.py:30]: :100:

File: octobot_trading/exchanges/connectors/ccxt_exchange.py [hunk 2]
```diff
@@ -171,12 +163,24 @@ def _unauthenticated_exchange_fallback(self, err):
         self.client = self._get_unauthenticated_exchange()
 
     def _get_unauthenticated_exchange(self):
-        return self.exchange_type({
+        return self.exchange_type(self._get_client_config())
+
+    def _get_client_config(self, api_key=None, secret=None, password=None):
```
Comment (Herklos) [inline octobot_trading/exchanges/connectors/ccxt_exchange.py:168]: Can you add a cython declaration like `_get_unauthenticated_exchange` please ?

File: octobot_trading/exchanges/abstract_exchange.py [hunk 3]
```diff
@@ -558,6 +558,12 @@ def get_max_handled_pair_with_time_frame(self) -> int:
         """
         return self.connector.get_max_handled_pair_with_time_frame()
 
+    def get_additional_connector_config(self):
```
Comment (Herklos) [inline octobot_trading/exchanges/abstract_exchange.py:561]: :+1:

**Review by Herklos — COMMENTED** [review]
:+1:

File: octobot_trading/exchanges/connectors/ccxt_exchange.py (hunk 2 above)
Comment (GuillaumeDSM) [reply octobot_trading/exchanges/connectors/ccxt_exchange.py:168]: sure :D

**Review by Herklos — APPROVED** [review]
:+1:

**Comment by GuillaumeDSM** [comment]
hollaex tests are passing with this ccxt version

**Comment by Herklos** [comment]
> hollaex tests are passing with this ccxt version

:+1:

**Pull request merged**